README.md

```diff
@@ -1 +1,65 @@
-# Goruchie.github.io
\ No newline at end of file
+# First-Capstone-Project ![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> This is a the first Microverse capstone project for HTML/CSS & Javascript module.
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Linters
+
+## Live Demo
+
+[Video Demonstration](https://www.loom.com/share/ebbe03099160458cabdf6e29c40b29a0)
+
+[Apex Legends Webinar 2022](https://goruchie.github.io/First-Capstone-Project/)
+ 
+## Getting Started
+
+**You can download, take a look and edit this project if you want to, but first make sure that you have the requirements to do it**
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Install Git.
+
+### you must have an account on git hub.
+
+### Install Visual Studio Code or any ide that you have.
+
+### Copy this repository link and clone it to your local machine.
+
+
+
+
+
+## Authors
+
+👤 **Marcos Gualtero Lourenzo**
+
+- GitHub: [githubhandle](@https://github.com/Goruchie)
+- Twitter: [@twitterhandle](https://twitter.com/Goruchie2)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/marcos-gualtero-a2aa35246/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Credit to [Cindy Shin in Behance](https://www.behance.net/adagio07) for the wonderful design
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] The link of `MIT` is broken please try to add a MIT file and and customize to your own as the previous reviewer stated